documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,91 @@
+# Get Bootchart from a Clear Container #
+
+The following steps describe how to get a bootchart from a Clear Containers
+container. Currently `systemd` is the default init process in the Clear Containers mini-os
+system. The Systemd project provides a tool called `systemd-bootchart` that
+collects the CPU load, disk load, memory usage, as well as per-process information
+from the os boot process.
+
+## Configure Clear Containers to get a bootchart ##
+
+To get a bootchart from a Clear Container, follow the steps below.
+
+1. Prepare your Clear Container image
+
+The file `data/cc-bootchart.conf` can be used get a bootchart graph.
```

Comment:
Is this optional? If it is, the sentence must be: "The file `data/cc-bootchart.conf` can be used to get a bootchart graph." If this file is necessary to get a bootchart graph, then it must be changed to: "Use the file `data/cc-bootchart.conf` to get a bootchart graph."